spec/shr_lab_panel_profiles.txt

```diff
@@ -0,0 +1,52 @@
+Grammar:	      DataElement 5.0
+Namespace:	    shr.lab
+Description:    "Profiles of common laboratory tests."
+Uses:           shr.core
+
+CodeSystem:     UCUM = http://unitsofmeasure.org
+CodeSystem:     SCT = http://snomed.info/sct
+
+EntryElement:   CBCWAutoDifferentialPanel
+Based on:       HeadlessLaboratoryPanel
+Description:    "57021-8 CBC W Auto Differential panel - Blood. 
+This profile is used to validate the panel, including the codes of the panel members, but not the panel members themselves. The panel members are stand-alone observations that must validate against the appropriate laboratory profile."
+                SubjectOfRecord is type PatientSubjectOfRecord
+                Code is LNC#57021-8
+                PanelMembers.Observation
+  includes 0..* CBCWAutoDifferentialPanelCoreElement
+  includes 0..* CBCWAutoDifferentialPanelNonCoreElement
+
+// Instances that satisfy CBCWAutoDifferentialPanelCoreElement will are required to satisfy the corresponding profiles in lab_2000 which determines the DataValue type or answer set for common LOINC codes
+        EntryElement:   CBCWAutoDifferentialPanelCoreElement
+        Based on:       SimpleQuantitativeLaboratoryObservation
+        Description:    "Must-support members of the CompleteBloodCountPanelByAutomatedCountPanel"
+                        SubjectOfRecord is type PatientSubjectOfRecord
+                        Code from CBCWAutoDifferentialPanelCoreElementVS    
+
+        EntryElement:   CBCWAutoDifferentialPanelNonCoreElement
+        Based on:       SimpleQuantitativeLaboratoryObservation
+        Description:    "Optional members of the CompleteBloodCountPanelByAutomatedCountPanel"
+                        SubjectOfRecord is type PatientSubjectOfRecord
+                        Code from CBCWAutoDifferentialPanelNonCoreElementVS
+
+EntryElement:   ComprehensiveMetabolic2000SerumOrPlasmaPanel
+Based on:       HeadlessLaboratoryPanel
+Description:    "24323-8 Comprehensive metabolic 2000 panel - Serum or Plasma.  This profile is used to validate the panel, including the codes of the panel members, but not the panel members themselves. The panel members are stand-alone observations that must validate against the appropriate laboratory profile."
+1..1            SubjectOfRecord is type PatientSubjectOfRecord
+                Code is LNC#24323-8
+                PanelMembers.Observation
+  includes 0..* CMP2000SerumOrPlasmaCoreElement
```

Comment:
One concern I have with this approach is that for every lab panel supported by mCODE, it forces implementers to read every analyte under the panel, separate them out to core and non-core and structurally map them into each sliced element.  Would this add more implementation burden and reduce adoption?